backend/src/platform/api/resolvers.py

```
@@ -166,48 +139,22 @@ def require_environment_access(
-    ]
-    require_resource_access_with_org(principal, env.created_by, creator_org_ids)
+    require_resource_access(principal_id, env.created_by)
     return env
 
 
-def require_run_access(session: Session, principal: Principal, run_id: str) -> TestRun:
+def require_run_access(session: Session, principal_id: str, run_id: str) -> TestRun:
+    """Check test run access and return run."""
     run = session.query(TestRun).filter(TestRun.id == run_id).one_or_none()
     if run is None:
         raise ValueError("run not found")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)